### PR TITLE
Merge Go App Template

### PR DESCRIPTION
**Pull request by jnsougata**

### CODE REVIEW

File: go-app/Discovery.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Go App Example"
+tagline: "A simple Go app running on Space"
+git: "https://github.com/deta/space-templates"
```
Comment (abdelhai) [inline go-app/Discovery.md:4]: should this be like this?
```suggestion
git: "https://github.com/deta/space-templates/tree/main/go-app"
```

**Review by abdelhai — APPROVED** [review]

File: go-app/Discovery.md (hunk 1 above)
Comment (lemonyte) [reply go-app/Discovery.md:4]: Does it accept any url? I was under the impression that it had to be in the format `https://domain/user/repo`.

Comment (abdelhai) [reply go-app/Discovery.md:4]: pretty sure any url for now. but this is still git

Comment (lemonyte) [reply go-app/Discovery.md]: Once every template has been merged, I will update all the markdown files for consistency. Hold off on merging #17 or similar ones until we work through the open template PRs.

**Pull request merged**